docker-compose.yml

```diff
@@ -6,5 +6,6 @@ services:
     container_name: tor-socks-proxy
     image: peterdavehello/tor-socks-proxy:latest
     ports:
+      - "127.0.0.1:53:53/udp"
```

Comment:
Do you mind to temporarily remove `/udp` for consistency, and send another PR to specify tcp and udp in both `docker-compose.yml` and `Dockerfile`? Many thanks!